### ISSUE TITLE
Office backstage Search: NVDA does not Read Search Suggestions when Arrow Up and Down the List of Document Suggestions.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13772]
<img width="957" alt="NVDA does not announce focused search suggestion list item" src="https://user-images.githubusercontent.com/93735775/172158348-40eb2157-159f-49d8-8f76-88d370b16078.png">

### Steps to reproduce:
1. Turn on NVDA.
2. Open Word Application and open blank file.
3. Navigate to Ribbon using f6 key.
4. Navigate to File tab using arrow key and activate it.
5. Backstage will be appeared.
6. Navigate to Search Edit in Home using tab key and enter some text in search edit field.
7. Suggestion will be appeared and navigate using up/down arrow key.
8. Observe the issue with NVDA.

### Actual behavior:
NVDA does not read search suggestions when arrow up and down the list of document suggestions. Screen reader user will be impacted, they will be confused and think there is no suggestion list and screen reader focus are stuck in search edit field if NVDA does not read search suggestions while navigating suggestions.

### Expected behavior:
Screen reader should announce the focused search suggestion list item when user navigate the list item using up/down arrow key.

### System configuration
Windows 11 Enterprise

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version 2022.1

#### Windows version:
Windows 11 
OS build: Version 21H2 Build 22000.675

#### Name and version of other software in use when reproducing the issue:
Word Version: 2206 Build 15330.20020

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Version 2020.4. issue is also reproducible with this NVDA version.

**Comment by josephsl** [comment]
Hi,

Technical: Office Backstage search suggestions interface elements are not tagged as proper search field and/or suggestions list items. UIA layout invalidated event is fired whenever suggestions list changes. Resolving this will require identifying uIA Automation Id, class names, and other properties for Office Backstage controls and mark them as proper overlay classes (the foundation is there in NVDA, but it comes down to overlay class chooser).

Thanks.
